pkg/scalers/azure_eventhub_scaler.go

```diff
@@ -171,6 +171,8 @@ func (scaler *azureEventHubScaler) GetUnprocessedEventCountInPartition(ctx conte
 		err = errors.Unwrap(err)
 		if stErr, ok := err.(azblob.StorageError); ok {
 			if stErr.ServiceCode() == azblob.ServiceCodeBlobNotFound || stErr.ServiceCode() == azblob.ServiceCodeContainerNotFound {
+				fmt.Errorf("Blob container : %s not found to use checkpoint strategy, error : %s", scaler.metadata.eventHubInfo.BlobContainer, err)
+				fmt.Println("Getting unprocessed event count without checkpoint")
```

Comment:
This case is not a critical error, is more like an execution condition. 
Could you print those messages in V(1)?
```suggestion
				eventhubLog.V(1).Error(err, fmt.Sprintf("Blob container : %s not found to use checkpoint strategy, error : %s", scaler.metadata.eventHubInfo.BlobContainer, err))
				eventhubLog.V(1).Info("Getting unprocessed event count without checkpoint")
```

Reply:
do we need to print both error and info log message? Isn't just one (info on debug) enough? In case this error is not critical.

Reply:
I assume is not critical because we don't return the error, so I guess that it's another code path.
Merging both messages in one is good idea IMO

Reply:
I was also wondering about the fatality of this error. I see we move on and get the count of events using partition count, but the user seems uninformed of this or at least hasn't subscribed to this mechanism for scaling decision. Any thoughts on this?

Reply:
I'm not an expert in Azure EventHub and its internals.
Do you know about it @tomkerkhove @ahmelsayed ?

Reply:
I'm not sure I get the question, sorry :/ Can you elaborate on it please?

Reply:
wdyt about the fatality of this error?

Reply:
@tomkerkhove As the code suggests, if a blob container is not found we get the event count as the number of messages in the partition. But, if the user has mentioned a checkpoint strategy they meant to use the blob container and process event count, so is silently logging an error and moving on to get event count from partition right?